python/lib/core/dmod/core/dataset.py

```diff
@@ -16,6 +16,46 @@
 from .common.reader import Reader
 
 
+# TODO: (later) consider putting this in a different module, and also adding utils for doing the archiving
+class DataArchiving(PydanticEnum):
+    """
+    Supported archiving techniques that may be applied to the entire contents of a file-based dataset.
+
+    Types of whole-dataset archiving techniques that are supported for DMOD datasets.  Note that these represent ways to
+    archive **all** the data of a dataset, when the dataset itself requires archiving.  Datasets may also contain data
+    archive files as individual data items, and such archive files are not necessarily restricted to these types.
+    """
+    TAR = (1, ".tar")
```

Comment:
Partially at least because we'd have to change the file extensions for the zip-related values; right now they are all `.zip` for simplicity.  Whether it makes sense to do that here is debatable, but my initial thought was that this was consistent with real-world usage.